CompetitionTemplate/src/competition/subsystems/drive/commands/MonitorDefenseTraversalModule.java

```diff
@@ -0,0 +1,57 @@
+package competition.subsystems.drive.commands;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.PropertyManager;
+import xbot.common.properties.StringProperty;
+import edu.wpi.first.wpilibj.Timer;
+
+@Singleton
+public class MonitorDefenseTraversalModule {
+    
+    public enum DefenseState {
+        NotOnDefense,
+        OnDefense,
+        RecentlyOnDefense
+    }
+
+    double timeOfRecentPitchEvent;
+    DoubleProperty defenseTraversalTime;
+    DoubleProperty defenseTraversalPitchThreshold;
+    StringProperty defenseTraversalState;
+    
+    @Inject
+    public MonitorDefenseTraversalModule(PropertyManager propMan) {
+        timeOfRecentPitchEvent = -99999999;
```

Comment:
Uhhhhh.... How about `Double.MIN_VALUE`? Or, if you know that only positive numbers can be returned, -1.